.travis.yml

```diff
@@ -1,5 +1,16 @@
 sudo: false
 language: rust
+
+before_install:
```

Comment:
Could you move this after `matrix` and before `install`? It's more coherent, as before_install happens on Travis right before install.

Reply:
Looks good, but this change is still missing @sahitpj